### PR TITLE
Fixes content alignment issues in homepage

### LINKED ISSUE #1173: home page contents not aligned properly
**Describe the bug**
- *Launch Simulator* and *For Teachers* buttons not aligned properly
- The *new_homepage_sketch* image is not aligned with corresponding content.

**To Reproduce**
1. Go to beta.circuitverse.org
2. See error

**Expected behavior**
The forementioned buttons must be aligned properly and the image must be aligned with it's contents.

**Screenshots**
Screenshot from 2020-03-14 10-26-51

Screenshot from 2020-03-14 11-17-42

**Additional context**

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by Nitish145**
Fixes #1173 

#### Describe the changes you have made in this PR -
- Removes x-axis margin in Launch simulator row.
- Use consistent margin of 15px on left side.

### Screenshots of the changes (If any) -
Screenshot from 2020-03-14 11-01-29

### BEFORE
Screenshot from 2020-03-14 11-17-42

### AFTER
Screenshot from 2020-03-14 11-24-32

### CODE REVIEW

**Comment by Nitish145** [comment]
@nitin10s @aayush-05 reviews please :)

**Comment by Nitish145** [comment]
@nitin10s @aayush-05 reviews please :)

**Comment by Nitish145** [comment]
### BEFORE
Screenshot from 2020-03-15 14-55-31

### AFTER
Screenshot from 2020-03-15 14-55-14

Does this work @nitin10s ?

**Comment by aayush-05** [comment]
Don't use flexbox, we are already using row, col structure. Instead I would prefer simply using a media query.

**Comment by aayush-05** [comment]
@Nitish145 just fix the x-alignment in this pr. #1147 is already open for the top margin

**Comment by Nitish145** [comment]
X-alignment is indirectly linked to stray padding @aayush-05

**Comment by Nitish145** [comment]
@aayush-05 I think replacing row-col structure here with flexbox makes sense because we will be needing extra *class* defs to center the content in respective divs..

**Comment by Nitish145** [comment]
Removing stray paddings also fixes #1146

**Comment by Nitish145** [comment]
@nitin10s @aayush-05 reviews please :)

**Comment by aayush-05** [comment]
> @aayush-05 I think replacing row-col structure here with flexbox makes sense because we will be needing extra _class_ defs to center the content in respective divs..

I personally prefer following the complete container-row-col structure rather than using some of container classes and then combining it with some user-defined flexbox. That is what the official documentation recommends ([Bootstrap Docs](https://getbootstrap.com/docs/4.4/layout/overview/)).
Talking about center aligning, if you want to center-align everything just put it in the container class, no need to do it for every element. In this case, it was done for explicitly the image because the text needed to be left aligned.
Due to my preferences, I can be a little biased here so I will leave it to @tachyons @nitin10s

**Comment by Nitish145** [comment]
@nitin10s any more changes here ??

**Comment by nitin10s** [comment]
Please rebase @Nitish145

**Comment by Nitish145** [comment]
Done! Please check @nitin10s

**Comment by nitin10s** [comment]
Resolve merge conflicts.

**Comment by Nitish145** [comment]
@nitin10s done! Please check :)

File: config/master.key [hunk 1]
```diff
@@ -0,0 +1 @@
+b2661ba33d44a917476a6c2e42129e2b
```
Comment (tachyons) [inline config/master.key]: Remove this

Comment (Nitish145) [reply config/master.key]: Done @tachyons !!

**Pull request merged**